typings/viewbox.d.ts

```diff
@@ -0,0 +1,24 @@
+// Type definitions for viewbox 1.0
+// Project: https://github.com/tracespace/viewbox
+// Definitions by: Mike Cousins <https://mike.cousins.io>
+
+declare module 'viewbox' {
+  namespace viewbox {
+    type ViewBox = Array<number>
+
+    interface Rectangle {
+      x: number
+      y: number
+      width: number
+      height: number
+    }
+
+    function create(): ViewBox
+    function add(box: ViewBox, addend: ViewBox): ViewBox
+    function scale(box: ViewBox, scale: number): ViewBox
```

Comment:
This fails lint in @typescript-eslint/plugin@>=1.4.0, see typescript-eslint/typescript-eslint#316